adapters/nftkey/index.js

```diff
@@ -0,0 +1,139 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const Avalanche = require("../../sdk/avalanche");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const BigNumber = require("bignumber.js");
+
+class NFTKey {
+    constructor() {
+        this.name = "nftkey";
+        this.symbol = "AVAX";
+        this.token = "avax";
+        this.protocol = "avalanche";
+        this.block = 6421617;
+        this.contract = "0x1A7d6ed890b6C284271AD27E7AbE8Fb5211D0739";
```

Comment:
For EVM chains all addresses must be lowercased (provider contract, buyer address, seller address and so on)

Reply:
@liutaurasjanuskas  Updated

Reply:
@mariya169 Seller also has to be lowercased.

Reply:
from the test log i've seen before, i think i don't have to make seller / buyer address to lowercase manually, which should always be lowercase.

Reply:
@liutaurasjanuskas yeah, i just checked the code, it should be set `toLowercase` manually. Changed now.